.github/workflows/build-deb.yaml

```diff
@@ -0,0 +1,117 @@
+name: Build debian packages
+
+on:
+  push:
+    branches:
+      - main
+    tags:
```

Comment:
We don’t want to restrict on tags, everything that is push to main or opened as PR should have that passing.